src/services/spectral-agent-service.ts

```diff
@@ -131,12 +138,20 @@ export class SpectralAgentService {
 
     item.rule.severity = this.mapToNewSeverity(item.rule.severity)
     item.finding = item.finding.toLowerCase()
+    item.labelDisplayName = item.rule.name
+
     this.findingsAggregations[item.rule.severity] += 1
     const isIac = item.metadata.tags.includes(FindingType.iac)
+    const isOss = item.metadata.tags.includes(FindingType.oss)
+
     let findingTypeResults
     if (isIac) {
       findingTypeResults = this.findings[FindingType.iac]
       this.findingsAggregations[FindingType.iac] += 1
+    } else if (isOss) {
+      item.labelDisplayName = `${item.rule.name} - ${item.rule.id}`
+      findingTypeResults = this.findings[FindingType.oss]
+      this.findingsAggregations[FindingType.oss] += 1
     } else {
```

Comment:
there is no other 3rd option, but I think that for better readability, and future engines that will be added, it is better to calculate `isSecrets` like `isIac` and add it to the last condition.

Reply:
We don't have a specific tag for secrets, and I kept it as is until we added property means which engine run.